au4/src/au3wrap/internal/au3audiodevicesmanager.h

```diff
@@ -9,13 +9,26 @@
 
 #include "libraries/lib-utility/IteratorX.h"
 #include "libraries/lib-strings/wxArrayStringEx.h"
+#include "playback/iaudiodevicesmanager.h"
 
 namespace au::au3 {
-class Au3AudioDevicesManager
+class Au3AudioDevicesManager : public playback::IAudioDevicesManager
 {
 public:
     void init();
 
+    QStringList getAudioOutputDevices() const override;
+    QString currentAudioOutputDevice() const override;
+    void setAudioOutputDevice(const QString& deviceName) override;
+    muse::async::Notification audioOutputDeviceChanged() const override;
+
+    void handleDeviceChange() override;
+
+    QStringList getHosts() const override;
```

Comment:
I believe the term `audio api` is more understandable than `host` in this case.

Reply:
but maybe I'm wrong :)

Reply:
Just followed au3 style at first, but audioApi sounds more understandable for me as well, changed :+1: